test/blocks/sidenav/sidenav.test.js

```diff
@@ -27,16 +27,37 @@ describe('Sidenav', () => {
     window.fetch = sinon.stub().callsFake(() => mockedTaxonomy());
   });
 
-  it('does create nice categories sidenav', async () => {
-    const sidenavEl = document.querySelector('.sidenav.categories');
-    await init(sidenavEl);
-    const newRoot = document.querySelector('merch-sidenav');
-    expect(newRoot.title).to.equal("REFINE YOUR RESULTS");
+  const testCategorySidenav = async (selector, expectedNumber, expectedChildrenNumber) => {
+    const sidenavEl = document.querySelector(selector);
+    const newRoot = await init(sidenavEl);
+    expect(newRoot.tagName).to.equal('MERCH-SIDENAV');
+    expect(newRoot.title).to.equal('REFINE YOUR RESULTS');
     const items = newRoot.querySelectorAll('sp-sidenav-item');
-    expect(items.length).to.equal(24);
+    expect(items.length).to.equal(expectedNumber);
```

Comment:
`expectedNumber`,  maybe `expectedItemCount`?
In such tests, I like inline values as it is easier to read.